onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -237,16 +240,16 @@ static void DumpOrtModelAsJson(const std::string& model_uri) {
   std::ofstream(model_uri + ".json") << json;
 }
 */
-/* The full build was causing the following error because the graph node array has some empyt (blank) node at some indices for certain ORT designs
-       onnx runtime exception : Satisfied, but should not be : node == nullptr
-       session_state.cc : 814 onnxruntime::SessionState::LoadFromOrtFormatCan't find node with index 4. Invalid ORT format model.
-  The bug was due to loading an ORT format model in a full build, allowing optimizers to run, but trying to use the saved kernel information.
-  As the optimizer removed some leaving gaps in the graph node vector.
-  The build has been fixed in InferenceSession code. The following test case to catch this error.
-*/
 
+/* 
+Validate we don't run optimizers on an ORT format model in a full build. The optimizers will remove nodes,
+which will create a mismatch with the saved kernel information and result in a runtime error. 
+We could take steps to handle this scenario in a full build, but for consistency we choose to not run optimizers 
+on any ORT format model.
+*/
 TEST(OrtModelOnlyTests, ValidateOrtFormatModelDoesNotRunOptimizersInFullBuild) {
-  const std::basic_string<ORTCHAR_T> ort_file = ORT_TSTR("mnist.onnx.ort");
+  // we have tests that use a pre-created minst.onnx.ort, so make the naming for the unit test generated file clearer
+  const std::basic_string<ORTCHAR_T> ort_file = ORT_TSTR("testdata/mnist.onnx.test_output.ort");
   SaveAndCompareModels("testdata/mnist.onnx", ort_file);
```

Comment:
renaming to clarify which ort files are test input (for the minimal build tests), which ones are created by running tests, and so that all models are in the same build/test output directory.